src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(new File(toFileName), makeReport(readLinesFromFile(fromFileName)));
+    }
+
+    private StringBuilder readLinesFromFile(String fromFileName) {
+        File fileFrom = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileFrom))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
```

Comment:
It's better to use " " here. You can easily make "split" by it in makeReport method
```suggestion
                stringBuilder.append(value).append(" ");
```